JsonRead/src/main/java/weather/Main.java

```diff
@@ -0,0 +1,18 @@
+package weather;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+        String city = "Berlin,de";
+        String uRLString = "https://api.openweathermap.org/data/2.5/" +
+                "weather?q=" + city + "&units=metric&appid=335838461bf85dd808aef371792bb429";
+        
+        ReadFrom read = new ReadFromInternet();
+        ForecasterService forecasterService = new ForecasterService();
```

Comment:
If ReadFrom needs the ForecastService, the pass ForecastService to the ReadFromInternet as parameter to constructor.